samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
+##################
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
+
+Overview
+********
+
+The sample demonstrates usage of NCP target on the MCU to communicate it with user WPAN Network Daemon, allowing to create for example Thread Border Router.
+It is important to mention that, according to NCP architecture, MCU part needs to cooperate with user higher layer process to altogether establish complete Thread Border Router.
+Recommended solution is to use docker version of OTBR (OpenThread Border Router) and WPAN Network Daemon tool (wpantund), working under Linux operating system, for such purpose (for more information go there: `OpenThread Border Router`_).
```

Comment:
What is the OTBR docker? Please explain or provide a link to the explanation.

Reply:
I rephrased it, because OTBR is just exemplary solution and I should have been explain this. Anyway I'm not sure what do you mean. Should I clarify what is docker? I wrote what is OTBR in brackets and the link for explanation with install tutorial is also attached.